cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js

```diff
@@ -342,3 +342,331 @@ describe("Card - Refund flow test", () => {
+    });
+
+    context("Fully Refund Card-ThreeDS payment flow test Create+Confirm", () => {
+
+        it("create+confirm-payment-call-test", () => {
+          console.log("confirm -> " + globalState.get("connectorId"));
+          let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+          cy.createConfirmPaymentTest( createConfirmPaymentBody, det,"three_ds", "automatic", globalState);
+        });
+
+        it("Handle redirection", () => {
+            let expected_redirection = confirmBody["return_url"];
+            cy.handleRedirection(globalState, expected_redirection);
+          })
+    
+          it("retrieve-payment-call-test", () => {  
+            cy.retrievePaymentCallTest(globalState);
+          });
+  
+          it("refund-call-test", () => {
+              let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+              cy.refundCallTest(refundBody, 6540, det, globalState);
+          });
+      
+      });
+
+    context("Partially Refund Card-ThreeDS payment flow test Create+Confirm", () => {
+
+        it("create+confirm-payment-call-test", () => {
+          console.log("confirm -> " + globalState.get("connectorId"));
+          let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+          cy.createConfirmPaymentTest( createConfirmPaymentBody, det,"three_ds", "automatic", globalState);
+        });
+        
+        it("Handle redirection", () => {
+            let expected_redirection = confirmBody["return_url"];
+            cy.handleRedirection(globalState, expected_redirection);
+          })
+
+         it("retrieve-payment-call-test", () => {  
+          cy.retrievePaymentCallTest(globalState);
+        });
+
+        it("refund-call-test", () => {
+            let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+            cy.refundCallTest(refundBody, 3000, det, globalState);
+        });
+
+        it("refund-call-test", () => {
+            let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+            cy.refundCallTest(refundBody, 3000, det, globalState);
+        });
+
+        it("sync-refund-call-test", () => {
+            let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+            cy.syncRefundCallTest(det, globalState);
+        });
+    
+    });
+
+    context("Card - Full Refund for fully captured 3DS payment", () => {
+
+        it("create-payment-call-test", () => {
+            let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+            cy.createPaymentIntentTest(createPaymentBody, det, "three_ds", "manual", globalState);
+        });
+
+        it("payment_methods-call-test", () => {
+            cy.paymentMethodsCallTest(globalState);
+        });
+
+
+        it("confirm-call-test", () => {
+            console.log("confirm -> " + globalState.get("connectorId"));
```

Comment:
Remove this unwanted log from all places

Reply:
Done.